lib/oli_web/live/delivery/student/lesson/components/one_at_a_time_question.ex

```diff
@@ -0,0 +1,513 @@
+              </div>
+            </div>
+            <div class="flex justify-center w-full min-h-[84px] items-center">
+              <button
+                :if={!selected_question.submitted}
+                phx-click="submit_selected_question"
+                phx-target={@myself}
+                phx-value-attempt_guid={selected_question.state["attemptGuid"]}
+                phx-value-question_id={"question_#{selected_question.number}"}
+                disabled={!selected_question.answered}
+                class={[
+                  "h-[30px] px-5 py-2.5 rounded-md shadow justify-center items-center gap-2.5 inline-flex opacity-90 text-right text-base text-white font-['Open Sans'] leading-normal whitespace-nowrap",
+                  if(selected_question.answered,
+                    do: "bg-[#0062f2] font-semibold",
+                    else: "bg-[#9d9d9d] font-semibold "
+                  )
+                ]}
+              >
+                Submit Response
+              </button>
+              <div :if={selected_question.submitted} class="activity w-full p-2 px-10">
+                <div role="question points feedback" class="flex justify-end mb-2.5">
+                  <span class="text-[#8e8e8e] text-xs font-normal font-['Open Sans'] leading-[18px]">
+                    Points:
+                  </span>
+                  <span class="ml-1 text-[#5e5e5e] text-xs font-semibold font-['Open Sans'] leading-[18px]">
+                    <%= question_points(selected_question) %> / <%= total_question_points(
+                      selected_question
+                    ) %>
+                  </span>
+                </div>
+                <div role="question feedback" class="activity-content">
+                  <%= OliWeb.Common.React.component(
+                    @ctx,
+                    "Components.Evaluation",
+                    %{
+                      attemptState: selected_question.state,
+                      context: selected_question.context,
+                      showExplanation: false
+                    },
+                    id: "activity_evaluation_for_question_#{selected_question.number}",
+                    container: [class: "flex flex-col w-full"]
+                  ) %>
+                </div>
+              </div>
+            </div>
+            <.references ctx={@ctx} bib_app_params={@bib_app_params} />
+          </div>
+        </div>
+        <div role="questions footer" class="w-[1170px] pl-[189px] mb-32 py-8 flex justify-between">
+          <button
+            phx-click={JS.dispatch("click", to: "#question_#{selected_question.number - 1}_button")}
+            disabled={selected_question.number == 1}
+            id="previous_question_button"
+            class="w-[117.45px] h-[30px] px-5 py-2.5 bg-white rounded-md shadow border justify-center items-center gap-2.5 inline-flex"
+          >
+            <div class="justify-end items-center gap-2 flex">
+              <svg
```

Comment:
That was my first approach but I found other arrow icons in that module (up_arrow, right_arrow, down_arrow, previous_arrow, etc,etc) that didn't match the one I needed. 
But, thinking it again, although the icon might not end up being re-used by other templates, it will improve the readability of this one.